File: app/src/main/java/com/aaron/yespdf/filepicker/AbstractAdapter.java
package com.aaron.yespdf.filepicker;

import androidx.recyclerview.widget.RecyclerView;

/**
 * @author Aaron aaronzzxup@gmail.com
 */
abstract class AbstractAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {

    abstract void selectAll(boolean selectAll);

    abstract boolean reset();
}

File: app/src/main/java/com/aaron/yespdf/filepicker/AbstractAdapter.kt
package com.aaron.yespdf.filepicker

import androidx.recyclerview.widget.RecyclerView

/**
 * @author Aaron aaronzzxup@gmail.com
 */
abstract class AbstractAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder?>() {
    abstract fun selectAll(selectAll: Boolean)
    abstract fun reset(): Boolean
}